addons/auth_signup/controllers/main.py

```diff
@@ -42,6 +42,12 @@ def web_auth_signup(self, *args, **kw):
         if 'error' not in qcontext and request.httprequest.method == 'POST':
             try:
                 self.do_signup(qcontext)
+
+                # set user to public if they were not signed in by do_signup (mfa enabled)
```

Comment:
Nitpicking: In general I prefer all comments start with an uppercase and do not overflow 80 char per line :+1: